src/main/java/it/gov/pagopa/payment/service/payment/common/CommonAuthServiceImpl.java

```diff
@@ -97,13 +99,11 @@ protected AuthPaymentDTO invokeRuleEngine(String trxCode, TransactionInProgress
             sendAuthPaymentNotification(trx);
 
         } else if (trx.getStatus().equals(SyncTrxStatus.AUTHORIZED)) {
-            throw new ClientExceptionWithBody(
-                    HttpStatus.FORBIDDEN,
+            throw new TransactionAlreadyAuthorizedException(
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor